src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,16 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				if (ev.changedTouches === undefined) {
+					this._trigger('touchStart');
```

Comment:
@rovolution I actually did that originally, but wasn't sure if that would cause any accidental event from happening that's bound to slideStart. Did you prefer slideStart?